packages/fxa-auth-server/lib/error.js

```diff
@@ -200,6 +201,9 @@ AppError.translate = function(request, response) {
   if (response instanceof AppError) {
     return response;
   }
+  if (OauthError.isOauthRoute(request && request.route.path)) {
+    return OauthError.translate(response);
+  }
```

Comment:
a little shim for converting oauth errors. Unfortunately we've got overlapping `errno`s with different meanings between auth and oauth